app/webpacker/javascript/controllers/community_activity_component_controller.js

```diff
@@ -51,7 +80,17 @@ export default class extends ApplicationController {
           this.saveDraftButtonTarget.setAttribute('disabled', '')
```

Comment:
Given what we discussed this morning about Draft button being enabled when they have some data, will this not cause problems as it going to disable that button when they have a missing value?

Reply:
Ignore this, just me being stupid and not noticing the code fold in the github review